app/elements/coder/challenge/tasks/endpoints/canvas-game-task.html

```diff
@@ -2,12 +2,8 @@
 <link rel="import" href="../../../../../../bower_components/paper-button/paper-button.html">
 <link rel="import" href="../../../../../../bower_components/paper-material/paper-material.html">
 <link rel="import" href="../../../../../../bower_components/paper-toast/paper-toast.html">
-<link rel="import" href="/elements/coder/code-editor/code-editor.html">
-<link rel="import" href="/elements/coder/code-editor/language-selector.html">
-<link rel="import" href="/elements/common/services/template-service.html">
-<link rel="import" href="/elements/tasks/task-display.html">
 <link rel="import" href="../../../../../../bower_components/iron-flex-layout/classes/iron-flex-layout.html">
-
+<!--TODO(victorbalan) adapt this task to our current model-->
```

Comment:
yes..at least for now. I will deal with all the deprecated stuff sometime after these weeks.